doc/guides/modules.rst

```diff
@@ -231,7 +241,7 @@ Changes to existing modules
 #. Submit the changes using a pull request to an existing repository following
    the :ref:`contribution guidelines <contribute_guidelines>`.
 #. Submit a pull request changing the entry referencing the module into the
-   :zephyr_file:`west.yml` of the main Zephyr tree with the following
+   `west.yml`_ of the main Zephyr tree with the following
```

Comment:
I'd recommend leaving this as `` :zephyr_file:`west.yml` `` (as noted below, you shouldn't lock it to master)